taclib/container.py

```diff
@@ -350,11 +350,20 @@ def _wait_for_status(self, job, desired_status, timeout=15):
                         return False
             return True
         except ReadTimeoutError:
-            log = getLogger(__name__)
-            log.warning("Timeout while waiting for status %s! Pod had status:"
-                        " %s" % (desired_status, self._get_pod(job).status.phase))
+            status = self._get_pod(job).status.phase
+            self._warn_pod_status_timeout(desired_status, status)
             return False
 
+    @staticmethod
+    def _warn_pod_status_timeout(desired_status, status):
+        log = getLogger(__name__)
+        if (desired_status != "Running" or status != "Succeeded") \
```

Comment:
We don't want it to log when the desired status is 'running' and the pod status is different (like pending)?